tests/test_extract.py

```diff
@@ -0,0 +1,159 @@
+# input files are x1d, flt, corrtag, and counts
+
+from calcos.x1d import *
+
+
+def generate_fits_file(file):
+    """
```

Comment:
I wonder if it would make sense to move the file creation functions into a separate script and call those functions like you are calling calcos.x1d? This is because you call them in several test functions.